File: Ejercicios C/12.Operaciones.cpp
#include <iostream>
using namespace std;

int main() {
    // Variables para almacenar los números introducidos por el usuario
    double A, B;

    // Solicitar los números al usuario
    cout << "Ingrese el valor de A: ";
    cin >> A;
    cout << "Ingrese el valor de B: ";
    cin >> B;

    // Realizar operaciones
    double suma = A + B;
    double resta = A - B;
    double multiplicacion = A * B;
    double division = A / B;

    // Mostrar los resultados de las operaciones
    cout << "La suma de los números es: " << suma << endl;
    cout << "La resta de los números es: " << resta << endl;
    cout << "La multiplicación de los números es: " << multiplicacion << endl;
    cout << "La división de los números es: " << division << endl;

    // Comparar los valores de A y B
    bool igual = (A == B);
    cout << "¿El número " << A << " es igual a " << B << "? " << (igual ? "Sí" : "No") << endl;

    // Determinar el mayor y el menor de los dos números
    double mayor = (A > B) ? A : B;
    double menor = (A < B) ? A : B;

    // Mostrar el número mayor y menor
    cout << "El número mayor es: " << mayor << endl;
    cout << "El número menor es: " << menor << endl;

    return 0;
}

File: Ejercicios PHP/12.Operaciones.php
<?php
//En este caso se le pedira al usuario el dato de entrada definidos en dos variables.
echo("Ingrese el primer valor: ");
$A = (int)(fgets(STDIN));//int: define el valor de la variable como un numero.
echo("Ingrese el segundo valor: ");
$B = (int)(fgets(STDIN));
//Definimos las operaciones a realizar.
$suma = $A+$B;
echo("la suma de los numeros es: $suma\n");
$res = $A-$B;
echo("la resta de los numeros es: $res\n");
$multi = $A*$B;
echo("la multiplicacion de los numeros es: $multi\n");
if ($B != 0) {
    $div = $A / $B;
    echo("La división de los números es: $div\n");
} else {
    echo "No se puede dividir por cero.\n";
}

//Definimos una variable que compare el igual y el mayor entre A y B.
$igual = ($A == $B) ? 'sí' : 'no';//compara los valores y devuelve una respuesta textual.
echo "¿El número $A es igual a $B? $igual\n";

$mayor = ($A > $B) ? $A : $B;
$menor = ($A < $B) ? $A : $B;

echo "El número menor es: $menor\n";//Devuelve el valor numerico si el valor es verdadero.
echo "El número mayor es: $mayor\n";//Devuelve el valor numerico si el valor es falso.

//Para ejecutar en terminal: php "C:\Sena\Analisis y desarrollo de software\Ficha 3066478 segundo trimestre\Ejercicios PHP\12.Operaciones.php"
?>